test/e2e/aadpodidentity_test.go

```diff
@@ -712,6 +708,68 @@ var _ = Describe("Kubernetes cluster using aad-pod-identity", func() {
 	})
 })
 
+func collectLogs(podName, dir string) {
+	pods, err := pod.GetAllNameByPrefix(podName)
+	Expect(err).NotTo(HaveOccurred())
+	for _, p := range pods {
+		logFile := path.Join(dir, p)
+		cmd := exec.Command("bash", "-c", "kubectl logs "+p+" > "+logFile)
+		util.PrintCommand(cmd)
+		_, err := cmd.Output()
+		Expect(err).NotTo(HaveOccurred())
+	}
+}
+
+func collectPods(dir string) {
+	logFile := path.Join(dir, "pods")
+	cmd := exec.Command("bash", "-c", "kubectl get pods > "+logFile)
```

Comment:
can we do `kubectl get pods -o wide` so we also have the node information for all the pods?